day_two_guide/topics/api_service_status.adoc

```diff
@@ -24,6 +24,7 @@ oc get pod -n kube-system -o wide
 NAME                                               READY     STATUS    RESTARTS   AGE       IP            NODE
 master-api-myserver.com                            1/1       Running   0          7h        10.240.0.16   myserver.com/healthz
 
-$ curl -k https://myserver.com/healthz
+$ curl -k https://myserver.com:443/healthz <1>
```

Comment:
Change this line as below,
This must be reachable from client’s network:
$ curl -k https://master.example.com:443/healthz
ok

Reply:
@wangke19 thank you, I have now pushed those changes. @xingxingxia based on your earlier comment, did you want to make a further review?